go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/ElrondNetwork/arwen-wasm-vm/v1_4 v1.4.30
 	github.com/ElrondNetwork/concurrent-map v0.1.3
 	github.com/ElrondNetwork/covalent-indexer-go v1.0.4
-	github.com/ElrondNetwork/elastic-indexer-go v1.1.27-0.20211126141844-4d87b794c1a5
+	github.com/ElrondNetwork/elastic-indexer-go v1.1.28-0.20211208132212-2ac845160155
```

Comment:
not for this PR: we need a proper release

Reply:
For the moment let's keep the commit here, and when the `feat/SC-proc-improvements-NOV` is ready to be merged in `development` will make a proper release also on the `elastic-indexer-go` module. 
We have to validate that the indexer works in all the cases with this new branch.